src/components/OptionsMenu/OptionsMenu.tsx

```diff
@@ -18,12 +18,18 @@ function OptionsMenu(props: Props) {
     let { trackEvent } = useMatomo();
     let [anchorEl, setAnchorEl] = useState(null);
 
+    const getItemTag = (isItemPage: number) => {
+        return window.location.href.substr(isItemPage + 6).split('/')[0].split('?')[0];
+    }
+    
     let available: AvailableLinks[] = [];
-    const isItemPage = window.location.href.indexOf("/item/") > 0;
```

Comment:
isItemPage seems like a boolean. Let it be a boolean

Reply:
reverted